daemons/fenced/fenced_commands.c

```diff
@@ -3269,7 +3266,8 @@ handle_device_add_request(pcmk__request_t *request)
                          PCMK_EXEC_INVALID,
                          "Unprivileged users must register device via CIB");
     }
-    fenced_send_device_notification(op, &request->result, device_id);
+    fenced_send_device_notification(op, &request->result,
+                                    (dev == NULL)? NULL : ID(dev));
```

Comment:
It is just one more occurrence I found having a quick check but maybe a silent version of ID would still make sense.

Reply:
Yep or even a version of crm_element_value() that doesn't complain